lib/evm_database.rb

```diff
@@ -63,10 +63,10 @@ def self.seed(classes = nil, exclude_list = [])
     classes ||= PRIMORDIAL_CLASSES + (seedable_model_class_names - PRIMORDIAL_CLASSES)
     classes -= exclude_list
 
+    lock_timeout = ENV["SEEDING_LOCK_TIMEOUT"].to_i # default: 600
+    lock_timeout = 10.minutes if lock_timeout.zero?
```

Comment:
Maybe `lock_timeout = ENV["SEEDING_LOCK_TIMEOUT"] || 10.minutes`
Is there any other time that this is zero?